test/builtin/test_datentime.py

```diff
@@ -136,13 +136,13 @@ def test_private_doctests_datetime(str_expr, msgs, str_expected, fail_msg):
     [
         ##
         (
-            "TimeConstrained[Integrate[Sin[x]^100,x],.001]",
+            "TimeConstrained[Integrate[Sin[x]^1000,x],.001]",
```

Comment:
This should be in its own PR.  Also, the fact that we have to constantly adjust this suggests that we are doing this the wrong way.  

There is probably a way using Python's[ mock library ](https://docs.python.org/dev/library/unittest.mock.html )that would accomplish what we want to do more reliably and faster.

Reply:
Yes, I know, but since this is a draft branch, I take the chance to make experiments...

Reply:
Can't you set up a private fork of the repository and set up github workflows on that and do experiments this way? 

You tend to work in a very noisy way - use a the most watched repo of the organization, make lots of little commits that don't much public value nor have meaningful commit messages.   

I confess I will tend to do this myself on smaller lesser followed libraries, so yes, it is expedient. But noisy.

Reply:
OK, right, I'll do this in my github fork.